service/src/main/resources/quips/complibot/compliments.json

```diff
@@ -229,5 +240,126 @@
     "tags":[],
     "targetableText":"If there was a zombie apocalypse, I'd want resourceful people like [TARGET] in my camp.",
     "targetableSsml":"If there was a zombie apocalypse, I'd want resourceful people like [TARGET] in my camp."
+  },
+  {
+    "quipGroup":"INFECTIOUS_SMILE",
+    "text":"We better call the doctor, because your smile is infectious.",
```

Comment:
No it isn't, I have it in the Example Phrases section which will show up along with the skill.